mod_ci/controllers.py

```diff
@@ -152,12 +152,13 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
     with app.app_context():
         full_url = url_for('ci.progress_reporter', test_id=test.id,
                            token=test.token, _external=True, _scheme="https")
-    file_path = os.path.join(config.get('SAMPLE_REPOSITORY', ''), 'reportURL')
+    file_path = os.path.join(config.get('SAMPLE_REPOSITORY', ''),
+                             kvm_name + '/reportURL')
```

Comment:
please just add the kvm name as an argument to the join... join accepts multiple paths to be joined together.